doc/contributor_experience_team.rst

```diff
@@ -34,6 +34,10 @@
     <p>Chiara Marmo</p>
     </div>
     <div>
+    <a href='https://github.com/MaxwellLZH'><img src='https://avatars.githubusercontent.com/u/16646940?v=4' class='avatar' /></a> <br />
+    <p>Maxwell</p>
```

Comment:
```suggestion
    <p>Maxwell Liu</p>
```

Reply:
If the previous name was correct.

Reply:
@MaxwellLZH which do you prefer? You public github profile only has "Maxwell" and the author's script used by the release manager will always try to re-sync with the current state of your github profile.

We could add a special rule in `build_tools/generate_authors_table.py` to hard-code your name in the authors' list instead of attempting to sync with your github profile at each release.

Reply:
Hi @ogrisel I would prefer using `Maxwell Liu`. I've changed my github profile name to `Maxwell Liu` as well, so we don't need to add special rule :)

Reply:
Thanks!